libvips/foreign/dcrawload.c

```diff
@@ -197,6 +214,7 @@ vips_foreign_load_dcraw_set_metadata(VipsForeignLoadDcRaw *raw,
 	}
 	vips_image_set_int(image, VIPS_META_ORIENTATION, orientation);
 
+#if LIBRAW_COMPILE_CHECK_VERSION_NOTLESS(0, 21)
```

Comment:
This whole block requires libraw 0.21, see commit https://github.com/LibRaw/LibRaw/commit/9b946d94bd65d744538926c2b12957864e187b9b for reference.